User Management/src/main/java/com/usermanagement/model/Category.java

```diff
@@ -13,19 +13,19 @@
 @Entity
 @Table(name = "CATEGORIES", catalog = "XE")
 @Data
-public class Categories {
+public class Category {
 	
 	@Id
 	@Column(name = "id",length=32, unique = true, nullable = false, insertable = false, updatable = false)
 	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "SEQUENCE_NAME_CATEGORY")
 	@SequenceGenerator(name = "SEQUENCE_NAME_CATEGORY", sequenceName = "SEQUENCE_NAME_CATEGORY", allocationSize = 1, initialValue = 1)
-	private Integer id;
+	private int id;
 	@Column(name = "name",nullable = false, length = 50, unique=true)
 	private String name;
 	@Column(name = "description",nullable = false, length = 500)
 	private String description;
 	
-	public Categories(){}
+	public Category(){}

```

Comment:
If lombok is already being used, I recommend using the annotations @NoArgsConstructor and @AllArgsConstructor instead of manually declaring it.